src/main.cpp

```diff
@@ -1,6 +1,51 @@
-#include "testh.hpp"
+#include <cstdint>
+#include <iostream>
 
+#include "networking/Tcp.hpp"
+
+// Simple recv test for TCP interface
 int main() {
-    foo();
+    // Try to open a socket for listening
+    Tcp::ListenSocket liSock;
+    try {
+        liSock = Tcp::ListenSocket(1234);
```

Comment:
Can we introduce a config file here, so that we won’t hard code the port?

Reply:
main.cpp is just meant to be an example of how the TCP module might be used, but relegating data to config files and reading those files is definitely important. I would create a distinct branch/module for that.